apps/web/cypress/support/commands.ts

```diff
@@ -130,4 +132,29 @@ Cypress.Commands.add('clearDatabase', () => {
   return cy.task('clearDatabase');
 });
 
+Cypress.Commands.add('loginWithGithub', () => {
+  const githubUserEmail = Cypress.env('GITHUB_USER_EMAIL');
+  const githubPassword = Cypress.env('GITHUB_USER_PASSWORD');
+
+  cy.getByTestId('github-button').click();
+
+  return cy.origin(
+    'https://github.com',
+    { args: { githubUserEmail, githubPassword } },
+    ({ githubUserEmail, githubPassword }) => {
+      cy.get('#login_field').type(githubUserEmail);
+      cy.get('#password').type(githubPassword);
+      cy.get('input[type="submit"]').click();
+
+      cy.get('body').then(($body) => {
+        if ($body.find('#js-oauth-authorize-btn').length) {
+          cy.wait(2000).then(() => {
+            $body.find('#js-oauth-authorize-btn').trigger('click');
+          });
+        }
+      });
+    }
+  );
+});
```

Comment:
Added the new Cypress command which allows to Sign In/Up with a GitHub. It has the logic that after sign in checks if the authorize button is shown, and if then click on it.